software/docker/kafka-can-log/kafka_can_log/isobus.h

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
Concept question: Do we log CAN or ISOBUS?

I was under the impression we log CAN to kafka and then something else later parses the ISOBUS CAN messages into some ISOBUS specific avro message.

Reply:
We log raw CAN.

I used `isoblue.h` from @awlayton to parse PGNs from received CAN frames. (I know, it's hack too) The key of each serialized CAN message is its PGN. The idea is that from a Kafka consumer standpoint, we will use keys to filter out CAN messages from either `tra` or `imp` topic.